docs/src/use/configure/migration-guide.md

```diff
@@ -340,6 +340,7 @@ export default [
             sourceType: "module",
             globals: {
                 ...globals.browser,
+                ...globals.node,
```

Comment:
We should add `node: true` to `env` in the eslintrc configuration above because this shows "the same configuration in flat config".

Reply:
Whoops, forgot to push that commit. 🤦